spec/components/provider_interface/work_history_and_unpaid_experience_component_spec.rb

```diff
@@ -154,5 +156,29 @@
         end
       end
     end
+
+    context 'with unpaid experience that involved working with children' do
+      let(:volunteering_experiences) do
+        [build(:application_volunteering_experience,
+               role: 'TA',
+               working_pattern: 'Part time',
+               details: 'Supervising classroom',
+               working_with_children: true,
+               start_date: 7.years.ago,
+               end_date: nil)]
+      end
+      let(:work_experiences) { [] }
+
+      before { allow(volunteering_experiences.first).to receive(:application_form).and_return(application_form) }
+
+      it 'renders all experience' do
+        rendered = render_inline(described_class.new(application_form: application_form))
+
+        expect(rendered.text).to include "#{7.years.ago.to_s(:month_and_year)} - Present"
+        expect(rendered.text).to include 'TA - Part time (unpaid)'
+        expect(rendered.text).to include 'Supervising classroom'
+        expect(rendered.text).to include 'Worked with children'
```

Comment:
For volunteering, we show both the `details` and the worked with children status